### ISSUE TITLE
NonDeterministicRequireOrder should also apply to require_relative

### ISSUE THREAD

**Issue by gongfarmer** [rubocop/rubocop#10614]
This regards the cop Lint/NonDeterministicRequireOrder

Currently it does not flag code which uses require_relative instead of require, like this:

```
# require all ruby source files in ./update/
Dir.glob("#{__dir__}/update/*.rb") do |path|
  require_relative path
end
```

Current result:
```
$ rubocop lib/gpt/update.rb 
Inspecting 1 file
.

1 file inspected, no offenses detected
```

Desired result:
```
$ rubocop lib/gpt/update.rb 
Inspecting 1 file
W

Offenses:

lib/gpt/update.rb:6:1: W: [Correctable] Lint/NonDeterministicRequireOrder: Sort files before requiring them.
Dir.glob("#{__dir__}/update/*.rb") do |path|
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

1 file inspected, 1 offense detected, 1 offense auto-correctable
```
